docs/metrics.md

```diff
@@ -22,13 +22,63 @@ The following dimensions are automatically added to all metrics exported by the
 | `process.pid`            | The Java process identifier (PID).
 | `service`                | The value of the `service.name` resource attribute.
 
+## Manual instrumentation
```

Comment:
I've noticed that I forgot to document how to manually add metrics, so I added it right now - that's a feature that needs to be described in this release's docs.